flow-typed/npm/@emotion/core_vx.x.x.js

```diff
@@ -0,0 +1,18 @@
+// flow-typed signature: ae9bf87350fe0a51dd9e1860bc7e5187
+// flow-typed version: <<STUB>>/@emotion/core_v10.0.14/flow_v0.110.0
+
+/**
+ * This is an autogenerated libdef stub for:
+ *
+ *   '@emotion/core'
+ *
+ * Fill this stub out by replacing all the `any` types.
+ *
+ * Once filled out, we encourage you to share your work with the
+ * community by sending a pull request to:
+ * https://github.com/flowtype/flow-typed
+ */
+
+declare module '@emotion/core' {
+  declare module.exports: any;
```

Comment:
This and the styled-base stubs shouldn’t exist. Emotion already ships flow types

Reply:
there are flow type errors and there is no way to fix them
could be a temporary fix

Reply:
I just published a new version of @emotion/serialize which is the package where there were errors. Could you upgrade the emotion packages and remove these stubs?

Reply:
great! will do it now!